api.planx.uk/modules/webhooks/service/updateTemplatedFlowEdits/helpers.ts

```diff
@@ -19,11 +23,13 @@ export const transformDeltaToTemplatedFlowEditsData = (
       updatedNodeData[updatedKey] = data[nodeId]?.["data"]?.[updatedKey];
     });
 
-    const updatedEdges = nodeData?.edges;
+    // Always preserve a templated folders' "edges", even if it's been "emptied" out
+    const updatedEdges =
+      nodeData?.edges || nodeData?.type === ComponentType.InternalPortal;
```

Comment:
I _believe_ `ComponentType.InternalPortal` is the only edge case we need to explicitly account for here because it's valid for it to be completely empty. Upload and Label `rules` and Question or Checklist `options` cannot be "saved" if empty arrays.

Any others I might be forgetting?